### PR TITLE
Feature/robot  ge newgift

### PR DESCRIPTION
**Pull request by skristem**
AA W-039584
Adding a new test case file that tests the functionality of New gift button on Contact and Account records.
Added:robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot
Updated:robot/Cumulus/resources/GiftEntryPageObject.py
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Review by deepmalamehta — APPROVED** [review]

File: robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Setup Test Data
+    [Documentation]      Creates the organiation account and household contact record
+    ...                  along with getting dates required for test.
+    &{account} =         API Create Organization Account    Name=${faker.company()}
+    ...                  BillingStreet=${faker.street_address()}
+    ...                  BillingCity=${faker.city()}
+    ...                  BillingState=California
+    ...                  BillingPostalCode=${faker.postcode()}
+    Set suite variable   &{ACCOUNT}
+    ${date} =            Get Current Date         result_format=%Y-%m-%d
```
Comment (rjanjanam) [inline robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot:25]: Please capitalize all variables consistently

File: robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    ...                                  Amount=150.0
+    ...                                  CloseDate=${DATE}
+    ...                                  StageName=Closed Won
+    &{payment} =  API Query Record       npe01__OppPayment__c      npe01__Opportunity__c=${newopp_id}
+    Verify Expected Values               nonns                          npe01__OppPayment__c    ${payment}[Id]
+    ...                                  npe01__Payment_Amount__c=150.0
+    ...                                  npe01__Payment_Date__c=${DATE}
+    ...                                  npe01__Paid__c=True
+
+Create Gift For Account Using New Gift Button
```
Comment (rjanjanam) [inline robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot:65]: Did you want the two tests in the same robot file ?

**Review by rjanjanam — APPROVED** [review]
Looks good

**Review by rhakeshj — APPROVED** [review]

File: robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot (hunk 2 above)
Comment (skristem) [reply robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot:65]: they are both testing the same feature but on different objects and independent so thought to keep them together

File: robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot (hunk 1 above)
Comment (skristem) [reply robot/Cumulus/tests/browser/gift_entry/new_gift_functionality.robot]: thanks for catching, updated it.

**Review by codexmarcel — COMMENTED** [review]
@davidmreed rolled up what would've been a separate PR into this one by merging my branch into it. For the sake of completeness, requesting review scoped to the unpackaged metadata.

**Review by davidmreed — APPROVED** [review]

**Pull request merged**